public/app/features/provisioning/File/FileStatusPage.tsx

```diff
@@ -87,9 +87,18 @@ function ResourceView({ wrap, repo, repoRef, tab }: Props) {
   const { t } = useTranslate();
 
   const tabInfo = [
-    { value: TabSelection.File, label: 'File (from repository)' },
-    { value: TabSelection.Existing, label: 'Existing (from grafana)' },
-    { value: TabSelection.DryRun, label: 'Dry Run (result after apply)' },
+    {
+      value: TabSelection.File,
+      label: t('provisioning.resource-view.tab-info.label.file-from-repository', 'File (from repository)'),
+    },
+    {
+      value: TabSelection.Existing,
+      label: t('provisioning.resource-view.tab-info.label.existing-from-grafana', 'Existing (from grafana)'),
```

Comment:
```suggestion
      label: t('provisioning.resource-view.tab-info.label.existing-from-grafana', 'Existing (from Grafana)'),
```
Nit: an issue with the original text.

Reply:
Thanks, fixed this one and the other case below :+1: